spec/features/food_search_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe '/foods', :vcr do
+  it 'should go to /foods' do
+    visit '/'
+    fill_in 'q', with: 'tumeric'
+    click_on 'Search'
+    expect(page).to have_current_path('/foods',ignore_query: true)
+  end
+
+  it 'should have info on page' do
+    visit '/'
+    fill_in 'q', with: 'tumeric'
+    click_on 'Search'
+    expect(page).to have_content('Your query found 432 results for tumeric')
+    expect(page).to have_content("Brand Owner: ")
+    expect(page).to have_content('GTIN/UPC: ')
+    expect(page).to have_content('Description: ')
+    expect(page).to have_content('Ingredients: ')
+    # count elememts on page only ten
+  end
```

Comment:
I would write a test that aligns with the spec and the query parameter of sweet potatoes first. It's also a good idea to test for the dynamic values that will show up in the results. I usually test for the values at the front and back of the array if I'm iterating in a view.